examples/openshift-origin/cleanup.sh

```diff
@@ -15,11 +15,39 @@
 # limitations under the License.
 
 # Cleans up resources from the example, assumed to be run from Kubernetes repo root
-
+echo
+echo
 export OPENSHIFT_EXAMPLE=$(pwd)/examples/openshift-origin
 export OPENSHIFT_CONFIG=${OPENSHIFT_EXAMPLE}/config
-rm -fr ${OPENSHIFT_CONFIG}
-cluster/kubectl.sh delete secrets openshift-config
-cluster/kubectl.sh stop rc openshift
-cluster/kubectl.sh delete rc openshift
-cluster/kubectl.sh delete services openshift
+
+echo "===> Removing OpenShift:"
+kubectl delete secrets openshift-config
+kubectl delete rc openshift
+kubectl delete services openshift
+echo
+
+echo "===> Removing etcd:"
+kubectl delete rc etcd
+kubectl delete services etcd
+echo
+
+echo "===> Removing etcd-discovery:"
+kubectl delete rc etcd-discovery
+kubectl delete services etcd-discovery
+echo
+
+echo "===> Removing local files:"
+rm -rf ${OPENSHIFT_CONFIG}
+rm ${OPENSHIFT_EXAMPLE}/secret.json
+echo
+
+echo "===> Restoring changed YAML specifcations:"
+if [ -f "${OPENSHIFT_EXAMPLE}/etcd-controller.yaml.bak" ]; then
+	rm ${OPENSHIFT_EXAMPLE}/etcd-controller.yaml
+	mv -v ${OPENSHIFT_EXAMPLE}/etcd-controller.yaml.bak ${OPENSHIFT_EXAMPLE}/etcd-controller.yaml
+else
+	echo "No changed specifications found."
+fi
+echo
+
+echo Done.
```

Comment:
this whole script could be made easier if we created OpenShift in a dedicated Namespace.

In that way, we could just delete the Namespace.

Can you update that?